platform/server/src/main/java/org/motechproject/server/util/MessageBrokerPingUtil.java

```diff
@@ -0,0 +1,115 @@
+package org.motechproject.server.util;
+
+import org.apache.activemq.ActiveMQConnectionFactory;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import javax.jms.Connection;
+import javax.jms.DeliveryMode;
+import javax.jms.Destination;
+import javax.jms.JMSException;
+import javax.jms.Message;
+import javax.jms.MessageConsumer;
+import javax.jms.MessageProducer;
+import javax.jms.Session;
+import javax.jms.TextMessage;
+
+/**
+ * Created by atish on 9/7/15.
+ */
+public final class MessageBrokerPingUtil {
+
+    private static final Logger LOGGER = LoggerFactory.getLogger(MessageBrokerPingUtil.class);
+
+    private ActiveMQConnectionFactory factory;
+
+    private Connection connection;
+
+    private Session session;
+
+    private Destination destination;
+
+    private MessageProducer producer;
+
+    private MessageConsumer consumer;
+
+
+    private MessageBrokerPingUtil() {
+
+    }
+
+    private static class SingletonHelper {
+        private static final MessageBrokerPingUtil INSTANCE = new MessageBrokerPingUtil();
+    }
+
+    public static MessageBrokerPingUtil getInstance() {
+        return SingletonHelper.INSTANCE;
+    }
+
+    private void connect(final String queueUrl) throws JMSException {
+        factory = new ActiveMQConnectionFactory(queueUrl);
+        LOGGER.info(String.format("Connecting with broker using connector url %s.", queueUrl));
```

Comment:
Goes for all logs in this class - don't use String.format, use {} and let the logger do the replacing